lstchain/scripts/lstchain_mc_sensitivity.py

```diff
@@ -40,31 +43,32 @@
                     help = 'path to protons DL1 file')
 parser.add_argument('--input-file-gamma-dl2-cuts', '--gd2-cuts', type = str,
                     dest = 'dl2_file_g_cuts',
-                    help = 'path to reconstructed gammas dl2 file used to caculate the sensitivity')
+                    help = 'path to reconstructed gammas dl2 file used to calculate the sensitivity')
 parser.add_argument('--input-file-proton-dl2-cuts', '--pd2-cuts', type = str,
                     dest = 'dl2_file_p_cuts',
-                    help = 'path to reconstructed protons dl2 file used to caculate the sensitivity')
+                    help = 'path to reconstructed protons dl2 file used to calculate the sensitivity')
 parser.add_argument('--input-file-gamma-dl2-sens', '--gd2-sens', type = str,
                     dest = 'dl2_file_g_sens',
-                    help = 'path to reconstructed gammas dl2 file used to caculate the optimized cuts' 
-                    'to be applied to the the sensitivity')
+                    help = 'path to reconstructed gammas dl2 file used to calculate the optimized cuts' 
+                    ' to be applied to the sensitivity')
 parser.add_argument('--input-file-proton-dl2-sens', '--pd2-sens', type = str,
                     dest = 'dl2_file_p_sens',
-                    help = 'path to reconstructed protons dl2 file used to caculate the optimized cuts'
-                    'to be applied to the the sensitivity')
+                    help = 'path to reconstructed protons dl2 file used to calculate the optimized cuts'
+                    ' to be applied to the sensitivity')
 
 args = parser.parse_args()
 
 
 def main():
-    ntelescopes_gamma = 4
-    ntelescopes_protons = 4
-    n_bins_energy = 20  #  Number of energy bins
-    n_bins_gammaness = 11  #  Number of gammaness bins
+    ntelescopes_gamma = 1
+    ntelescopes_protons = 1
+    n_bins_energy = 10  #  Number of energy bins
+    n_bins_gammaness = 10  #  Number of gammaness bins
     n_bins_theta2 = 10  #  Number of theta2 bins
     obstime = 50 * 3600 * u.s
-    noff = 5
-
+    noff = 3
```

Comment:
`noff` should be 5 to calculate the sensitivity using the CTA-style

Reply:
Where do you have this info from? Would be good to link a source if it is written somewhere that CTA expects 5 off positions.

Reply:
I could not find it in the IRF webpage (https://www.cta-observatory.org/science/cta-performance/), although I may be missing it.
In any case, one can quote the [Monte Carlo performance studies for the site selection of the Cherenkov Telescope Array](https://www.sciencedirect.com/science/article/abs/pii/S0927650517300087)paper:
"_A ratio of the off-source to on-source exposure of 5 is considered_"

Reply:
OK good thanks